### DISCUSSION TITLE
Selfhealing for WDIO

### DISCUSSION THREAD

**Discussion by bajipogadadanda** [webdriverio/webdriverio#13619] [General]
Hi, I would like to know if WDIO has any service/capability to implement self-healing feature (like Katalon has this capability  ) to handle the dynamic locators.
It will be really helpful if anyone has attempted it for WDIO.

**Comment by christian-bromann** [comment]
@Bapugit WebdriverIO currently does not have such capabilities afaik. If you like to get involved help to implement this I would love to connect. There is a contributor stipend to expense folks that provide such contributions.
